client/packages/requisitions/src/ResponseRequisition/DetailView/RelatedDocumentsSection.tsx

```diff
@@ -34,7 +34,7 @@ export const RelatedDocumentsSection: FC = () => {
   const { shipments } = useResponse.document.fields('shipments');
 
   const getTooltip = (createdDatetime: string, username?: string) => {
-    let tooltip = t('messages.outbound-shipment-created-on', {
+    let tooltip = t('messages.supplier-return-created-on', {
```

Comment:
I think this might be the only required change - this should be the inbound shipment one instead of outbound 😅